server/.env

```diff
@@ -17,4 +17,4 @@ ENABLE_EMAIL='false'
 APPLICATION_SECRET='myclassroom'
 LOG_LEVEL=''
```

Comment:
I'm still not sure I agree with renaming the `.env.example` file to `.env` as it might create issues later down the line. So it might be a good idea to get the opinion of others in the group to see if it is a concern or not

Also just a personal preference but IMO the 'log_level' should be set to debug by default and can be turned off if necessary.

Reply:
Renamed `.env` back to `.env.example` and added instructions to quickly rename it via a shell. Also set the log level to debug by default.